### PR TITLE
Number Conversion in different languages 

### LINKED ISSUE #865: Number Convertions in different languages
Can i work on it ??

### PR DESCRIPTION
**Pull request by mruna7**
Fixes #865 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
-  a new conversion is added
- 

#### Languages Used:
- java
- 

#### Files Added:
- 
- Decimal_To_Binary.java

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by singh-shreya6** [comment]
Please fix the PR template and mention the issue number

**Comment by mruna7** [comment]
> Please fix the PR template and mention the issue number

yes mam

File: Number_Conversion/Java/Decimal_to_Binary [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import java.util.Scanner;
+
+/**
+ * @author mrunal
+ *
+ */
+public class DecimalToBinary {
+       
+	static void Decimal_to_binary(int decimal)
+	{   int i=0;
```
Comment (Vaibhavraj10) [inline Number_Conversion/Java/Decimal_to_Binary]: Indentation. Declare in new line.

File: Number_Conversion/Java/Decimal_to_Binary [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		int binary[]=new int[100];
```
Comment (Vaibhavraj10) [inline Number_Conversion/Java/Decimal_to_Binary]: Give space between =

File: Number_Conversion/Java/Decimal_to_Binary [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+		while(decimal>0)
```
Comment (Vaibhavraj10) [inline Number_Conversion/Java/Decimal_to_Binary]: space between >

File: Number_Conversion/Java/Decimal_to_Binary [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+		{
+			binary[i]=decimal%2;
```
Comment (Vaibhavraj10) [inline Number_Conversion/Java/Decimal_to_Binary]: space between = and %

File: Number_Conversion/Java/Decimal_to_Binary [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+			decimal=decimal/2;
```
Comment (Vaibhavraj10) [inline Number_Conversion/Java/Decimal_to_Binary]: space between =  and /

File: Number_Conversion/Java/Decimal_to_Binary [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+			i++;
+		}
+		System.out.println("binary representation");
+		for(int j=i;j<=i;j--)
```
Comment (Vaibhavraj10) [inline Number_Conversion/Java/Decimal_to_Binary]: space between '=' , ';' , '<='

File: Number_Conversion/Java/Decimal_to_Binary [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+			System.out.print(binary[j]);
+	}
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		Scanner sc=new Scanner(System.in);
```
Comment (Vaibhavraj10) [inline Number_Conversion/Java/Decimal_to_Binary]: space between =

File: Number_Conversion/Java/Decimal_to_Binary [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+		System.out.println("Enter a decimal number");
+		int decimal=sc.nextInt();
```
Comment (Vaibhavraj10) [inline Number_Conversion/Java/Decimal_to_Binary]: space between =

File: Number_Conversion/Java/Decimal_to_Binary [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+		Decimal_to_binary(decimal);
+		
+
```
Comment (Vaibhavraj10) [inline Number_Conversion/Java/Decimal_to_Binary]: Remove these empty lines.

File: Number_Conversion/Java/Decimal_to_Binary [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+	}
+
+}
```
Comment (Vaibhavraj10) [inline Number_Conversion/Java/Decimal_to_Binary]: Add sample input and output. Do not blindly tick everything in the PR template.

File: Number_Conversion/Java/Decimal_to_Binary [hunk 11]
```diff
@@ -22,8 +22,8 @@ public class DecimalToBinary
 			i++;
 		}
 		System.out.println("binary representation");
-		for(int j = i ; j < = i ; j--)
-			System.out.print(binary[j]);
+		for(int j = i ; j <= i ; j--)
```
Comment (Vaibhavraj10) [inline Number_Conversion/Java/Decimal_to_Binary]: Just change the spacing as `j = i; j <= i; j-- `

**Comment by somya-kapoor** [comment]
@mruna7 Are you working on it?

**Comment by mruna7** [comment]
> @mruna7 Are you working on it?

yess mam !!done

**Comment by somya-kapoor** [comment]
@mruna7 Decimal_to_Binary in Java is already present in the repository.

**Comment by mruna7** [comment]
> @mruna7 Decimal_to_Binary in Java is already present in the repository
it was not there when i started.
now what to do??

**Pull request closed without merging**